Source/SelfService/Web/customer/view.tsx

```diff
@@ -0,0 +1,139 @@
+            '& .MuiOutlinedInput-input': {
+                color: 'white'
+            },
+            '& .MuiInputLabel-root': {
+                color: 'white'
+            },
+            '& .MuiOutlinedInput-root .MuiOutlinedInput-notchedOutline': {
+                color: 'white',
+                borderColor: 'white'
+            },
+            '&:hover .MuiOutlinedInput-input': {
+                color: 'white'
+            },
+        },
+        stepIcon: {
+            'color': '#3B3D48',
+            '&.MuiStepIcon-active': {
+                color: '#6678F6'
+            },
+            '&.MuiStepIcon-completed': {
+                color: '#6678F6'
+            },
+            '&.MuiStepIcon-active .MuiStepIcon-text': {
+                fill: '#B3BBFB'
+            },
+            '&.MuiStepIcon-root .MuiStepIcon-text': {
+                fill: '#93959F'
+            }
+        }
+    })
+);
+
+export const View: React.FunctionComponent<any> = (props) => {
+    const classes = useStyles();
+    const { enqueueSnackbar } = useSnackbar();
+
+    const { customerId } = useParams();
+    const [configLoaded, setConfigLoaded] = useState(false);
+    const [customerLoaded, setCustomerLoaded] = useState(false);
+    const [config, setConfig] = useState({} as Studio);
+    const [customer, setCustomer] = useState({} as Customer);
+    const [toggleConfig, setToggleConfig] = useState(false);
+
+    useEffect(() => {
+        getCustomers()
+            .then(customers => {
+                const customer = customers.find(customer => customer.id === customerId);
+                if (!customer) {
+                    enqueueSnackbar('No customer with the given id found', { variant: 'error' });
+                    return;
+                }
+
+                setCustomer(customer);
+                setCustomerLoaded(true);
+            }).catch((error) => {
+                console.log(error);
+                enqueueSnackbar('Failed getting customers data from the server', { variant: 'error' });
+            });
+    }, []);
+
+    useEffect(() => {
+        getStudioConfig(customerId)
+            .then(config => {
+                setConfig(config);
+                setConfigLoaded(true);
+            }).catch((error) => {
+                console.log(error);
+                enqueueSnackbar('Failed getting customers studio config from the server', { variant: 'error' });
+            });
+    }, [toggleConfig]);
+
+    if (!configLoaded || !customerLoaded) {
+        return null;
+    }
+
+    return (
+        <>
+            <h1>Customer {customer.name}</h1>
+            <div className={classes.root}>
+                <h3>Customer Id: {customerId}</h3>
+                <h3>Build overwrite: {String(config.buildOverwrite)}</h3>
+                <h3>Can create applications: {String(config.canCreateApplication)}</h3>
+                <h3>Disabled environments: {config.disabledEnvironments}</h3>
```

Comment:
Can we list this, as it is an array of strings.

Reply:
Made it into an unsorted list.